adapter/src/dbt/include/fal/macros/teleport_duckdb.sql

```diff
@@ -6,3 +6,8 @@
     CREATE OR REPLACE TABLE {{ relation }} AS
         SELECT * FROM '{{ url }}'
 {%- endmacro %}
+
+{% macro duckdb__copy_from_s3(relation, url) -%}
+    CREATE OR REPLACE TABLE {{ relation }} AS
+        SELECT * FROM read_parquet('{{ url }}');
+{%- endmacro %}
```

Comment:
I think we could change this to `duckdb__copy_from_parquet` and use it for both local and S3, and delete the `duckdb__copy_from` one.